<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroceryList</class>
 <widget class="QMainWindow" name="GroceryList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>852</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>393</width>
     <height>852</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>393</width>
     <height>852</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QStackedWidget" name="groceryft">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>-2</y>
      <width>393</width>
      <height>852</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>393</width>
      <height>852</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>393</width>
      <height>858</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="grocerymain">
     <property name="styleSheet">
      <string notr="true">background : none;
</string>
     </property>
     <widget class="QPushButton" name="addNewList">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>760</y>
        <width>87</width>
        <height>96</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/buttons/images/addbutton.png);
border: none;
box-shadow: none;
background-repeat: no-repeat;
background-position: center;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="grocerybg2">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-2</y>
        <width>393</width>
        <height>852</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/background/images/grocerybackground.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>108</y>
        <width>377</width>
        <height>680</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
background:none;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>377</width>
         <height>680</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background : none;

border: none;
background-color:transparent;
</string>
       </property>
       <widget class="QWidget" name="grocery_widget_list" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>3</y>
          <width>373</width>
          <height>670</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: none;
background-color:transparent;
</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="deleteButton">
      <property name="geometry">
       <rect>
        <x>315</x>
        <y>49</y>
        <width>42</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="grocerybackbutton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>41</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
box-shadow: none;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>79</y>
        <width>393</width>
        <height>788</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/background/images/bg.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>grocerybg2</zorder>
     <zorder>deleteButton</zorder>
     <zorder>grocerybackbutton_2</zorder>
     <zorder>label_4</zorder>
     <zorder>scrollArea</zorder>
     <zorder>addNewList</zorder>
    </widget>
    <widget class="QWidget" name="grocerylist">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QPushButton" name="addNewItem">
      <property name="geometry">
       <rect>
        <x>228</x>
        <y>755</y>
        <width>160</width>
        <height>89</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
box-shadow: none;
background-image: url(:/buttons/images/ADD.png);
background-repeat: no-repeat;
background-position: center;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="grocerybg">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-2</y>
        <width>393</width>
        <height>852</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>393</width>
        <height>852</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>393</width>
        <height>852</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/background/images/Grocery.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="grocerybackbutton">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>51</y>
        <width>39</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
box-shadow: none;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="grocery_list_name">
      <property name="geometry">
       <rect>
        <x>61</x>
        <y>46</y>
        <width>298</width>
        <height>48</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: none;
background-color: transparent;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="plainText">
       <string>Grocery List 1</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>192</y>
        <width>393</width>
        <height>657</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>393</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>395</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
background:none;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>393</width>
         <height>657</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>393</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>393</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:transparent;
background:none;

border: none;</string>
       </property>
       <widget class="QWidget" name="item_widget" native="true">
        <property name="geometry">
         <rect>
          <x>1</x>
          <y>2</y>
          <width>393</width>
          <height>675</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>393</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:transparent;
background:none;
</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="total_price">
      <property name="geometry">
       <rect>
        <x>196</x>
        <y>109</y>
        <width>162</width>
        <height>47</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="DeleteButton">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>785</y>
        <width>42</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/buttons/images/delete.png</normaloff>:/buttons/images/delete.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>166</y>
        <width>47</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Item</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>199</x>
        <y>168</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Quantity</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>307</x>
        <y>166</y>
        <width>47</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Price</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>178</y>
        <width>386</width>
        <height>788</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/background/images/bg.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>grocerybg</zorder>
     <zorder>grocerybackbutton</zorder>
     <zorder>total_price</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>label_5</zorder>
     <zorder>scrollArea_2</zorder>
     <zorder>grocery_list_name</zorder>
     <zorder>addNewItem</zorder>
     <zorder>DeleteButton</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
